janus_core/md.py

```diff
@@ -471,11 +488,20 @@ def _write_traj(self) -> None:
                 append=append,
             )
 
+    def _write_final_state(self) -> None:
+        """Write the final system state."""
+        if not self.restart_stem:
+            restart_file = f"{self.file_prefix}-T{self.temp}-final.xyz"
+        write(restart_file, self.struct, write_info=True, columns=None)
```

Comment:
Currently left the final state as ```columns=None``` for the moment

Reply:
Sorry just to check, does `columns=None` write everything (as you had it before)?

Reply:
I am a bit confused about this according to [ASE](https://gitlab.com/ase/ase/-/blob/master/ase/io/extxyz.py?ref_type=heads#L829) ```None``` writes this, so I think it writes less? 

```
fr_cols = (['symbols', 'positions']
                       + [key for key in atoms.arrays.keys() if
                          key not in ['symbols', 'positions', 'numbers',
                                      'species', 'pos']])
```

Reply:
None will write a standard xyz. we can consider adding columns as an option and user may select what wants to put in for final. but i will separate that as a different issue. we shall conclude this do not let perfect in the way of usable.